app/views/events/show.html.haml

```diff
@@ -0,0 +1,50 @@
+.row.col-sm-12
+  %h3
+    = @event.name
+  %h4
+    = "Organized by: #{@event.organizer_name}, scheduled at #{@event.formatted_scheduled_time}"
+  %h4
+    = @event.description
+%section.col-lg-3
+  - if @event.scheduled_hours
+    %p
+      %b
+        Duration:
+      = " #{@event.scheduled_hours} hours"
+  - if @event.capacity
+    %p
+      %b
+        Maximum participant:
+      = @event.capacity
+
+  - if current_user == @event.organizer
+    = link_to 'Edit', edit_user_event_path(@event.organizer, @event)
+    \|
+  = link_to 'Back', user_events_path(@user)
+
+%section.col-lg-9
+  .col-sm-12
+    %p 
+      %b
+        Address:
+      = @event.address
+  .col-sm-12#event-map{style: 'height: 400px;'}
+  :javascript
+    function initialize() {
```

Comment:
This `initialize` function will be in the global scope. Try to avoid adding to the global scope to minimize likelihood of naming collisions. Wrap with an IIFE to prevent adding `initialize` to the global scope, e.g.:

```javascript
(function() {
  function initialize() {
    ...
  }
})(); //The outer anonymous function is immediately invoked.
```